eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -238,7 +238,11 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
               }
           }
           // no need to go OFFLINE, we can directly switch to CLOSING
-          goto(CLOSING) using closing
+          if (closing.waitingSinceBlock > 1500000) {
```

Comment:
```suggestion
          if (closing.waitingSinceBlock > 1500000) {
            // we were using timestamps instead of block heights when the channel was created: we reset it *and* we use block heights
```

NB: This code will break in 15.7 year 😱

Reply:
I hope that in 15.7 years every eclair node will run a more recent version than eclair 0.5.1 and we'll have removed that code :smile: